py3status/modules/xrandr_toggle.py

```diff
@@ -88,7 +88,7 @@ def _xrandr(self):
                         monitors.append(source)
             return monitors
         except:
-            return self._error('error while parsing xrandr', None)
+            return self._error('error while parsing xrandr')
```

Comment:
This is still not the proper way to do this, I'm returning an error response instead of the monitors list.
Sorry for the mixup, I will fix this.